app/views/viewings/edit.html.erb

```diff
@@ -1,3 +1,7 @@
-<%= viewing.theatername %>
-<%= viewing.playtitle %>
-<%= viewing.date %>
\ No newline at end of file
+<h1>Edit Viewing</h1>
+
+<%= form_tag('/viewings/@viewing.id/update') do %>
```

Comment:
`form_tag` is a Ruby method. It accepts a string for its first argument.

Another example of a method like that is `puts`. E.g. `puts("Hi!")`.

What if you had this:

```ruby
name = "Sumeet"
puts("Hi!")
```

But instead of displaying just "Hi!", you wanted to display "Hi ____!" (with `name` filling in the blank)?

Report back please.